### DISCUSSION TITLE
Proposal to remove Mutable event listener for 0.3 final release

### DISCUSSION THREAD

**Discussion by yavko** [hyprland-community/hyprland-rs#22] [Polls]
Its very clunky, and impractical, I'd have to rewrite it to use multiple threads to make it practical, and that would take a while

**Comment by JustSimplyKyle** [comment]
You know what, removing it should be fine

**Reply by JustSimplyKyle** [reply]
but if you want to keep it, it's pretty great, since I don't want to do monitor grepping every single time

**Reply by yavko** [reply]
> but if you want to keep it, it's pretty great, since I don't want to do monitor grepping every single time

I can see the appeal and perf increase with it, and that's why I'm gonna rewrite it, but for the `0.3.1` release, as I'm releasing `0.3.0` tmrw, I'll prob have more details on discord
